pyzebra/comm_export.py

```diff
@@ -0,0 +1,134 @@
+
+def fill_string(string, total_lenght):
+    """fills string to tatal_length with whitespaces
+    :arg string - string to fill
+    :arg total_lenght - length of the final string
+    :return string of the lenght of total_lenght
+    """
+    white_spaces = " " * total_lenght
+    return_string = white_spaces[: -len(str(string))] + str(string)
+    return return_string
+
+
+def export_comm(data, path):
+    """exports data in the *.comm format
+    :param path: path to file + name
+    :arg data - data to export, is dict after peak fitting
+    """
+
+    if data["meta"]["indices"] == "hkl":

```

Comment:
This code needs to be deduplicated.
```
if data["meta"]["indices"] == "hkl":
    extension = ".comm"
elif data["meta"]["indices"] == "real":
    extension = ".incomm"

with open(str(path + extension), "w") as out_file:
    for keys in data["Measurements"]:
        if extension == ".comm":
            meas_number_str = fill_string(keys[1:], 6)
            ...
        else:
            meas_number_str = fill_string(keys[1:], 4)
            ...
etc.
```
Or something like that, probably the final form will be different depending on the approach you choose with the string interpolation.